src/responsibleai/registration_config.json

```diff
@@ -3,7 +3,7 @@
         "responsibleai-environment.yaml"
     ],
     "components" : [
-        "component_register_model.yaml",
+        "../common/component_register_model.yaml",
```

Comment:
Sneaky....

Reply:
yeah, I'm not convinced this is good "sneakiness" yet